WakaTime/Helpers/AppInfo.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  AppInfo.swift
```

Comment:
Is there a way to prevent Xcode adding these headers?

Reply:
I will remove them